docs/architecture/third-party/drupal/content-architecture.md

```diff
@@ -1 +1,14 @@
--- TO BE DEFINED --
\ No newline at end of file
+# Content architecture
+
+The OpenEuropa project does not enforce a strict content architecture but provides corporate solutions used by the European Commission.
+
+## Content types
+
+The [OpenEuropa Content](https://github.com/openeuropa/oe_content) component provides a growing number of corporate content types intended to harmonize the content architecture across the sites of the European Commission.
+
+Moreover, it provides RDF-based support for the corporate vocabularies published by the European Publication Office (OP).
+
+## Page building
+
+To support the building of pages with a variety of page-level elements, the [OpenEuropa Paragraphs](https://github.com/openeuropa/oe_paragraphs) component provides a number of Drupal paragraph types that can be used
+on any content type. Combined with the [OpenEuropa Theme](https://github.com/openeuropa/oe_theme), these are displayed using the [Europa Component Library (ECL)](https://ec.europa.eu/component-library) style guide.
```

Comment:
Can we change the ecl link with https://github.com/ec-europa/europa-component-library?

Reply:
Why?

Reply:
The link https://ec.europa.eu/component-library can change depending on the generator use.
But the Github is where the code and the documentation are based.

Reply:
Not really, that's actually the only unique URL among all of them as it's the project one.